README.md

```diff
@@ -37,3 +37,175 @@ docker run -it --rm \
   -v $(pwd)/boardswarm/share/server.conf:/etc/boardswarm/server.conf \
   ghcr.io/boardswarm/boardswarm:main
 ```
+
+## Manual Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
```

Comment:
Better to move in the boardswarm readme as that will also be distributed with the crate

Reply:
Would splitting out the server portion and client portion into the relevant READMEs and leaving just some pointers here to the individual parts make sense?

Reply:
Yeah that would be great; Generally the central README should be mostly pointers with most content in the respective READMEs so that if people enter through e.g. crates.io or anything else with the parts seperated everything relevant is there

Reply:
Hmm, been trying to do this, but basically to make the docs then flow, it goes from adding a "quick start guide" as I'd envisioned. to "marginal improvements to relatively lengthy documentation". Which wasn't exactly the intention...